Makefile

```diff
@@ -60,9 +60,13 @@ test: manifests generate fmt vet ## Run tests.
 
 ##@ Build
 
-build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager cmd/manager/main.go
-	go build -o bin/agent cmd/agent/main.go
+build: generate fmt vet build-manager build-agent
+
+build-manager:
+	CGO_ENABLED=1 go build -a -o bin/manager cmd/manager/main.go
```

Comment:
I am not sure whether we should hardcode it or you are able to set it from envvar